.github/workflows/ci.yml

```diff
@@ -70,6 +70,6 @@ jobs:
         run: |
           nim --version
           nimble --version
-          # nimble test
+          nimble test
           # nim c examples/ping.nim
```

Comment:
Do you think those comments are necessary?

Reply:
ping.nim and pong.nim are implemented (and uncommented) here https://github.com/status-im/nim-webrtc/pull/11.
So yeah, they're necessary. The alternative is to remove these two lines, but it won't change anything so I'll leave them commented.

Reply:
If they aren't implemented here, it's not necessary to add commented lines. They can be added to the appropriate PR. Additionally, could you please elaborate on what's the point of running those files on ci if they aren't tests?

Reply:
I see, but the step name says "Run tests" and additionally to its name, it's building examples, which is misleading. Let's create another task called "Build examples". The same for nim-libp2p.